tests/pr_test.go

```diff
@@ -114,8 +113,8 @@ func TestRunUpgradeSolutionInstances(t *testing.T) {
 		"kms_endpoint_type":                   "public",
 		"provider_visibility":                 "public",
 		"management_endpoint_type_for_bucket": "public",
-		"enable_platform_logs":                "false",
 		"enable_platform_metrics":             "false",
+		"logs_routing_tenant_regions":         []string{options.Region},
```

Comment:
@kierramarie Please remove this - right now we cannot create tenants in our tests because you can only have 1 tenant per region in an account. So until we write some automation to find a region with no tenant (tracked internally [here](https://github.ibm.com/GoldenEye/issues/issues/10676)), we have to just not create any in our tests, or they will clash and fail all the time (thats why the latest test run failed)